### PR TITLE
add the auto publicize config infra job

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -1117,6 +1117,51 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-merge-robot
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  decoration_config:
```
Comment (alvaroaleman) [inline ci-operator/jobs/infra-periodics.yaml:1123]: can you add the rehearsal label?

File: ci-operator/jobs/infra-periodics.yaml [hunk 2]
```diff
@@ -1117,6 +1117,49 @@ periodics:
...
+      - --dry-run=false
+      - --self-approve=true
+      - --github-login=openshift-merge-robot
+      - --git-name=openshift-bot
+      - --git-email=openshift-bot@redhat.com
+      - --publicize-config=./clusters/app.ci/publicize/_config.yaml
+      - --release-repo-path=./
+      - --whitelist-file=./core-services/openshift-priv/_whitelist.yaml
+      command:
+      - /usr/bin/autoperibolossync
```
Comment (stevekuznetsov) [inline ci-operator/jobs/infra-periodics.yaml:1148]: Is this right?

Comment (droslean) [reply ci-operator/jobs/infra-periodics.yaml:1148]: Damn copy pasta

File: ci-operator/jobs/infra-periodics.yaml (hunk 1 above)
Comment (droslean) [reply ci-operator/jobs/infra-periodics.yaml]: I don't think that we want to rehearse that. The rehearse job will open a new PR.

**Pull request merged**